README.md

```diff
@@ -41,15 +41,22 @@ import { TelnyxRTC } from '@telnyx/webrtc';
 
 To initialize the JavaScript SDK, you'll need to authenticate using a Telnyx Connection. You can access the Connection credentials in the [Telnyx Portal](https://portal.telnyx.com/#/app/connections).
 
+To use the `ringbackFile` make sure the option "Generate Ringback Tone" is disabled in your connection configuration on [Telnyx Portal](https://portaldev.telnyx.com/#/app/connections) Inbound tab.
+
 ```js
 // Initialize the client
 const client = new TelnyxRTC({
   // Required credentials
   login: username,
   password: password,
-  // Other options
-  // This file can be a wav/mp3 in your local public folder or you can host it in a CDN and pass just the URL, such as https://cdn.company.com/sounds/call.mp3
+  /* 
+    ringtoneFile - This file can be a wav/mp3 in your local public folder or you can host it in a CDN and pass just the URL, such as https://cdn.company.com/sounds/call.mp3
+   */
   ringtoneFile: './sounds/incoming_call.mp3',
+  /*
+    ringbackFile - Used when you disable Generate Ringback Tone to provide your own ringback sound - Enable Instant Ringback (180).
```

Comment:
Do you need to check the "Enable Instant Ringback (180)" option in the portal in order to use the ringback file?

Reply:
No, I will remove this.